docs/pages/yaml-reference.md

```diff
@@ -97,6 +93,20 @@ This documentation effectively reflects the Java object tree under [RawServiceSp
   * `strategy`
 
     This field isn't used! TODO(nickbp) remove.
+   
+  * `networks`
+
+    Allows the pod to join any number of virtual networks on the DC/OS cluster, however the only supported virtual network at present is the `dcos` overlay network. To have the pod join the overlay add the following:
+    ```
+    networks:
+      dcos:
+    ```
+    **N.B.** when a pod joins the overlay it has the following effects:
+     * The pod is given it's own IP address from the sub-net of the overlay belonging to the agent the pod is deployed to.
```

Comment:
When a pod joins the overlay network:

- The pod receives its own IP address from the subnet of the overlay belonging to the agent where the pod is deployed.